core/class.go

```diff
@@ -56,7 +56,28 @@ type Cairo1Class struct {
 	Program         []*felt.Felt
 	ProgramHash     *felt.Felt
 	SemanticVersion string
-	Compiled        json.RawMessage
+	Compiled        CompiledClass
+}
+
+type CompiledClass struct {
+	Bytecode        []*felt.Felt        `json:"bytecode"`
+	PythonicHints   json.RawMessage     `json:"pythonic_hints"`
+	CompilerVersion string              `json:"compiler_version"`
+	Hints           json.RawMessage     `json:"hints"`
+	EntryPoints     CompiledEntryPoints `json:"entry_points_by_type"`
+	Prime           string              `json:"prime"`
```

Comment:
Do we need to store the `Prime`? It will be the same for all classes.

Reply:
All Starknet contracts should use the same prime. I've [asked](https://t.me/sncorestars/30988) in the core stars channel just to be sure.

Reply:
I don't feel okay with one thing about this: the `Prime` will be used later in `vm.Class` to adapt from `core.CompiledClass` type to `feeder.CompiledClass` type, we use a constant string to set the prime field? And if the value changes in the future we will need to update that constant.

Reply:
I think that is ok. Converting to the feeder type and passing json to rust is a hack anyway.

Reply:
Hahaha, I'm convinced now ;v